GuideBuiltforUbuntuDebianGraftDockercontainer.md

```diff
@@ -0,0 +1,325 @@
+````bash
+Output:
+[sudo] password for username:
+````
+If your user is in the proper group and you entered the password correctly, the command that you issued with sudo should run with root privileges.
+
+## Install Docker as non-root user
+Note: if you are running as root please exclude any sudo references below:
+
+````bash
+sudo apt-get remove docker docker-engine docker.io
+sudo apt-get update
+sudo apt-get upgrade -y    # (for all promts just select keep as is) Optional
+````
+
+Install Docker dependencies (Copy entire command - all 6 lines and paste)
+
+````bash
+sudo apt-get install -y \
+apt-transport-https \
+ca-certificates \
+curl \
+gnupg2 \
+software-properties-common
+````
+ 
+ Add Docker’s official GPG key:
+
+````bash
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+````
+
+Test success with:
+
+````bash
+sudo apt-key fingerprint 0EBFCD88
+````
+
+Should return:
+````bash
+	pub   4096R/0EBFCD88 2017-02-22
+	      Key fingerprint = 9DC8 5822 9FC7 DD38 854A  E2D8 8D81 803C 0EBF CD88
+	uid                  Docker Release (CE deb) <docker@docker.com>
+	sub   4096R/F273FCD8 2017-02-22
+````
+
+## Add docker repo
+
+Stable repo:
+````bash
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic stable"
+````
+Edge repo:
+````bash
+sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu bionic edge"
+````
+
+Update apt again and install docker Community Edition:
+````bash
+sudo apt-get update
+sudo apt-get install -y docker-ce
+````
+
+Prepare Docker volume mount:
+````bash
+mkdir $HOME/.graft
+````
+
+## Download image and run container after with mounted volume 
+<B>(Check [Optional](https://github.com/mv1879/docs/blob/master/Dockers%20by%20Fez.md#optional) at botom of this guide before continuing)</B>
+:
+````bash
+sudo docker run --name graft -d -v $HOME/.graft:/root/.graft -p 28690:28690 -p 28680:28680 fez29/graftnoded-jagerman:Jagerman-Experiment_fez29
+````
+
+Use docker exec to login to the container as root:
+````bash
+sudo docker exec -ti graft /bin/bash
+````
+
+Note that graftnoded and graft_server are automatically started and restarted if they die by by supervisor - See optional if you want to disable/adjust behaviour. 
+(Check section in optional regarding downloading watch-only-wallets now if want to action)
+Check sync status:
+````bash
+graftnoded --testnet status
+````
+
+Check wallet address for Supernode:
+````bash
+graft-wallet-cli --wallet-file ~/.graft/supernode/data/stake-wallet/stake-wallet --password "" --testnet --trusted-daemon
+````
+Record seed and store safely especially on Mainnet (KEEP OFFLINE or written down and never reveal to anyone!)
+when in wallet type: seed and follow prompt (no password just press enter)
+
+Once graftnoded fully synced and stake in wallet, kill graft_server process to speed up process.  
+ Kill graft_server process:
+````bash
```

Comment:
These next few lines (using ps ax to get the pid and then kill) can be replaced with just:
```
killall -9 graft_server
```

Reply:
Noted will adjust